bin/Cargo.toml

```diff
@@ -47,7 +47,7 @@ dns-over-rustls = ["dns-over-tls", "dep:rustls", "hickory-server/dns-over-rustls
 dns-over-tls = []
 
 webpki-roots = ["hickory-server/webpki-roots"]
-native-certs = ["hickory-server/native-certs"]
+rustls-platform-verifier = ["hickory-server/rustls-platform-verifier"]
```

Comment:
I wonder if we should leave a backward compatible name for the feature here? Other option would be to add a comment when people using the old name have their compilation break.

Reply:
I think we can just call it out in the release notes? It breaks in a pretty obvious way. We've made enough changes that they're compilation will break anyway but evidence from upgraders so far seems to be that most downstream users can deal with it.

Reply:
Yes. I suppose the break will be obvious.